playback/service/src/main/java/de/danoeh/antennapod/playback/service/internal/LocalPSMP.java

```diff
@@ -182,8 +193,9 @@ private void playMediaObject(@NonNull final Playable playable, final boolean for
             } else {
                 throw new IOException("Unable to read local file " + media.getLocalFileUrl());
             }
-            UiModeManager uiModeManager = (UiModeManager) context.getSystemService(Context.UI_MODE_SERVICE);
-            if (uiModeManager.getCurrentModeType() != Configuration.UI_MODE_TYPE_CAR) {
+            androidAutoConnectionState.removeObserver(androidAutoConnectionObserver);
```

Comment:
Shouldn't this happen in `shutdown`? Otherwise it will only work for a single episode, not with continuous playback (I think)

Reply:
ready for a review again